R/datatype.R

```diff
@@ -152,67 +152,17 @@ DataType_new = function(str) {
 #' @examples
 #' # constructors are finally available via pl$... or pl$dtypes$...
 #' pl$List(pl$List(pl$Int64))
-DataType_constructors = list(
+DataType_constructors = function() {
```

Comment:
do we still need this function if we make one separate function per datatype?

Reply:
These functions need to remain as they are because of their special processing, such as being stored in two locations, like `pl$List` and `pl$dtypes$List`.